go/tools/builders/env.go

```diff
@@ -332,6 +339,21 @@ func absArgs(args []string, flags []string) {
 	}
 }
 
+// relativizePaths converts absolute paths found in the given output string to
+// relative, if they are within the PWD.
+func relativizePaths(output []byte) []byte {
+	dir, err := os.Getwd()
+	if dir == "" || err != nil {
+		return output
+	}
+	dirBytes := make([]byte, len(dir), len(dir)+1)
+	copy(dirBytes, dir)
+	if !bytes.HasSuffix(dirBytes, []byte{filepath.Separator})  {
+		dirBytes = append(dirBytes, filepath.Separator)
+	}
+	return bytes.ReplaceAll(output, dirBytes, nil)
```

Comment:
It looks like this still doesn't convert the working directory to `.`. For completeness, I think it ought to.

(I couldn't respond to the other comment chain; maybe there was a force push?)

Reply:
OK, I didn't add that because there didn't seem to be a use case and it complicates the code further, but it was easy enough to add.

Ah, I didn't know that a force push prevents response. In the future I'll avoid that